pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -60,6 +58,10 @@ const (
 	memoryAlertsGCInterval = 30 * time.Minute
 )
 
+// How long should we keep silences and notification entries on-disk after they've served their purpose.
```

Comment:
I'm not crazy about doing this, but I need to modify these numbers so that tests can take a reasonable amount of time. See https://github.com/prometheus/alertmanager/issues/2850 for another approach that could make testing faster without changing anything in the source file here.

One way to permit them to be modified for testing purposes would be to make them an argument to one of our functions - perhaps newAlertManager. I need to explore whether that would mess with all the wire magic that I don't yet understand.

Reply:
After submitting this I found that Prometheus does a pretty ugly hack of overriding time.Now in their own tests, but I like that hack better than making these non-const package variables and manipulating them during tests. I'll make a similar change to this PR